staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4131,8 +4134,86 @@ type Event struct {
 	// Type of this event (Normal, Warning), new types could be added in the future
 	// +optional
 	Type string `json:"type,omitempty" protobuf:"bytes,9,opt,name=type"`
+
+	// Time when this Event was first observed.
+	// +optional
+	EventTime metav1.MicroTime `json:"eventTime,omitempty" protobuf:"bytes,10,opt,name=eventTime"`
```

Comment:
IMO we make the old api extremely ugly this way. Technically adding elements to an old API is possible of course, but I have my doubt we want that. /cc @smarterclayton @deads2k opinions here?